### PR TITLE
check German translation for wrong wording

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
If needed, we can also add other language checks.

This commit has a faulty German string, so that we see that it fails.

TODO
- [x] check that it is working
- [ ] add more language checks?
- [x] roll back translation tests
- [ ] merge
- [ ] enable as required action

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
image

File: translations/client_de.ts [hunk 1]
```diff
@@ -6291,4 +6291,4 @@ Server antwortete mit folgendem Fehler: %2</translation>
         <translation>Ich habe etwas mit Ihnen geteilt</translation>
     </message>
 </context>
-</TS>
\ No newline at end of file
+</TS>
```
Comment (allexzander) [inline translations/client_de.ts:6294]: @tobiasKaminsky this must have a new line at the end, with that said, do you even need to change this file?

**Review by claucambra — COMMENTED** [review]
If you ask me, this doesn't scale. Are we going to provide test strings for every single language? What if the placeholder string changes remotely? Does the test fail and do we have to go back and change all the test strings again?

File: translations/client_de.ts (hunk 1 above)
Comment (tobiasKaminsky) [reply translations/client_de.ts:6294]: No.
I can revert it. Actually I have no idea where this comes from…

**Comment by allexzander** [comment]
> If you ask me, this doesn't scale. Are we going to provide test strings for every single language? What if the placeholder string changes remotely? Does the test fail and do we have to go back and change all the test strings again?
> 
> A better approach to this specific test might be to check for the language code (de_DE in this case) in the translation strings, as this is used in the placeholder strings. We're still open to having to change the test if the remote placeholder changes, but at least we can write one generic test checking if there are any strings containing the respective language code rather than having to hand-write tests for every single langauge

makes sense

**Review by claucambra — APPROVED** [review]

File: translations/client_de.ts (hunk 1 above)
Comment (claucambra) [reply translations/client_de.ts]: Fixed it

**Comment by allexzander** [comment]
@tobiasKaminsky ewww, we got that problem again, so, we've decided to have this as a temporary solution at least, automated commits from Transifex are giving us a lot of troubles :(

**Pull request merged**